app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -95,7 +95,7 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
 
   override fun onSaveInstanceState(): Parcelable {
     return PatientSummaryScreenSavedState(
-        super.onSaveInstanceState(),
+        super.onSaveInstanceState()!!,
```

Comment:
This doesn't seem to be necessary now since the `superSavedState` in `PatientSummaryScreenSavedState` is already nullable.

Reply:
Right, fixing this